### PR TITLE
this is my first task

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/1/style.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    border-bottom: 5px solid rgba(255, 255, 255, 0.692); 
+    color: rgba(255, 255, 255, 0.877);
+    
+}
+img {
+	margin: 8% 0 0 8%;
+	width: 310px;
+	position: absolute;
+}
+div.txt {
```
Comment (reiniergs) [inline cartaya/1/style.css]: div.text === .text ??? there are not the same I know but don't do that, avoid to use element selectors

File: cartaya/1/style.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+a {
+    font-size: 48px;
+    display: inline-block;
+    text-decoration: none;
+    border-bottom: 5px solid rgba(255, 255, 255, 0.692); 
+    color: rgba(255, 255, 255, 0.877);
+    
+}
+img {
```
Comment (reiniergs) [inline cartaya/1/style.css]: element selector !!

File: cartaya/1/style.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+body {
+    margin: 0;
+    font-family: sans-serif;
+}
+section {
+	height: 75vh;
+	background-color: #FFA600;
+}
+a {
```
Comment (reiniergs) [inline cartaya/1/style.css]: element selector

**Review by reiniergs — CHANGES_REQUESTED** [review]

File: cartaya/1/task.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    <section>
+        <div class="img">
+            <img src="1.jpg" alt="Cover Book" title="Book by Kawasaki">
+        </div>
+        <div class="txt">
+            <a href="#">
+                <span class="download">Download
+                <b class="for free">for free</b></span>
+            </a>
+            <br /><br />
```
Comment (TahimiLeonBravo) [inline cartaya/1/task.html]: Don’t  use ```<br />```  tag to increase the gap between lines of text; use the CSS margin  property

File: cartaya/1/task.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is my firs task from Leo">
+    <meta name="keywords" content="study,html5,css,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <title>Task#1</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <section>
+        <div class="img">
```
Comment (TahimiLeonBravo) [inline cartaya/1/task.html]: You need always use a convention to the names of all classes, it should create collisions.

File: cartaya/1/task.html [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            <br /><br />
+            <span>The ultimate visual guide:</span>
+            <br />
+            <span>168 hand-picked examples, 7 brillant</span>
+            <br />
+            <span>trends. inspiring, time-saving, and handy.</span>
+        </div>
+    </section>
+</body>
+</html>
```
Comment (TahimiLeonBravo) [inline cartaya/1/task.html]: You have to add a new line at the end of the file

File: cartaya/1/style.css [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+section {
+	height: 75vh;
+	background-color: #FFA600;
+}
+a {
+    font-size: 48px;
+    display: inline-block;
+    text-decoration: none;
+    border-bottom: 5px solid rgba(255, 255, 255, 0.692); 
+    color: rgba(255, 255, 255, 0.877);
```
Comment (TahimiLeonBravo) [inline cartaya/1/style.css]: You can define the color in the element container, this is an Inherited property you don’t need specify in each element inside container

File: cartaya/1/task.html (hunk 4 above)
Comment (TahimiLeonBravo) [inline cartaya/1/task.html]: The main reason for not using ```<br>``` is that it's not semantic. If you want two items in different visual blocks, you probably want them in different logical blocks.

**Pull request closed without merging**